ldi-core/ldes-client/tree-node-supplier/src/test/java/ldes/client/performance/csvwriter/CsvFile.java

```diff
@@ -49,7 +48,7 @@ private static void writeRecord(BufferedWriter bufferedWriter, String recordToWr
 			bufferedWriter.write(recordToWrite);
 			bufferedWriter.newLine();
 		} catch (IOException e) {
-			throw new StateOperationFailedException(e);
```

Comment:
Do we still need this then?

Reply:
No that is why we removed this. The CsvFile class itself is used for the performance test